FreifunkMemorandumofUnderstanding_it.md

```diff
@@ -0,0 +1,67 @@
+# Memorandum d'intesa di Freifunk
```

Comment:
Commento personale: io preferisco la forma _obiettivo_ con una sola _b_, però entrambe sono [accettate](http://treccani.it/enciclopedia/obiettivo-o-obbiettivo_%28La-grammatica-italiana%29), quindi possiamo anche lasciarla com'è se preferisci

Reply:
Thank you for reviewing my translation and helping getting merged ^^

I implemented each suggestion in a subsequent PR (except for _obiettivo_, being a matter of pure preference as you said).